### DISCUSSION TITLE
API Url multiple filter invalid

### DISCUSSION THREAD

**Discussion by indrazm** [pocketbase/pocketbase#1340] [Q&A] [answered]
Hey Guys,
I am trying to fetch my API URL with multiple filter but it give error.

```json
{
    "code": 400,
    "message": "Invalid filter parameters.",
    "data": {}
}
```

Here is my filter :
`?filter=(workspace='puzqw6i8f3f72o4'&&parent='')`

Am i setup wrongly ?
**Many Thanks!**

**Comment by ganigeorgiev** [comment, marked as answer]
If you are not using the JS or Dart SDKs, you may need to url encode the filter query parameter, eg.:

```
?filter=%28workspace%3D%27puzqw6i8f3f72o4%27%26%26parent%3D%27%27%29
```
